decentraland/constants/Colors.js

```diff
@@ -13,7 +13,7 @@ export default {
   noticeText: "#fff",
   backgroundColor: "#fff",
   textColor: "gray",
```

Comment:
This could be `textColor: SECONDARY_APP_COLOR` with `const SECONDARY_APP_COLOR = "gray"` or something like that

Reply:
Done. And I've also removed unused colors.

Reply:
Thanks! Resolving